Add WCF compat doc for bug#424922 (#970)
Microsoft/dotnet: Documentation/compatibility/wcf-accessbility-combobox-high-contrast-issue-in-svcTraceViewer-tool.md

## [svcTraceViewer ComboBox high contrast change]

### Scope
Edge

### Version Introduced
4.8
### Source Analyzer Status
NotPlanned

### Change Description

> In the Microsoft Service Trace Viewer tool, ComboBox controls were not displayed in the correct color in certain high contrast themes. The issue was fixed in 4.7.2 .NET Framework. However, due to .NET Framework SDK backward compatible requirement, it can't target 4.7.2 .NET Framework version, the fix was not visible to customer by default. An additional  fix is made in 4.8 to add the following configuration switch to svcTraceViewer.exe.config file

Reviewer: ```suggestion
In the [Microsoft Service Trace Viewer tool](~/docs/framework/wcf/service-trace-viewer-tool-svctraceviewer-exe.md), ComboBox controls were not displayed in the correct color in certain high contrast themes. The issue was fixed in .NET Framework 4.7.2. However, due to .NET Framework SDK backward compatibility requirements, the fix was not visible to customers by default. .NET 4.8 surfaces this change by adding the following [AppContext configuration switches](~/docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md) to the svcTraceViewer.exe.config file:
```

```xml
<AppContextSwitchOverrides value="Switch.UseLegacyAccessibilityFeatures=false;Switch.UseLegacyAccessibilityFeatures.2=false" />
```

### Recommended Action
   - How to opt out of change

>   If customer doesn't want to have the high contrast behavior change, they can work around by removing the following section from svcTraceViewer.exe.config.

Reviewer: ```suggestion
  If you don't want to have the high contrast behavior change, you can disable it by removing the following section from the svcTraceViewer.exe.config file:
```
